Docker/Runner/StateFile.php

```diff
@@ -84,23 +83,14 @@ public function createStateFile()
         $stateAdapter->writeToFile($stateFileName);
     }
 
-    public function storeStateFile()
+    public function storeState($currentState)
```

Comment:
místo `$currentState` by imho stačilo `$state`
`storeState` bych možná pojmenoval jako `saveState`?

Reply:
jakej je rozdil mezi save a store? :)

Reply:
no uz jsem to prejmenoval, ale ono je tam v te funkci jeste $previousState, tak jsem to zase vratil, jakoze $previousState a $currentState mi prijde lepsi nez $previousState a $state

Reply:
Tak v tom případě by to mělo bejt `currentState` a `newState`/`nextState`.

Reply:
(nebo aspoň takhle nějak to poslední dobou vídám)

Reply:
a jakjej je rozdil mezi previousState+currentState a currentState+nextState ? :)

Reply:
Funkčně žádnej, jenom se tahle jmenná konvence používá v Reactu, tak jsem si na to zvykl.